components/MentorRegistration/components/PersonDetails.js

```diff
@@ -59,8 +61,64 @@ function PersonDetails({
     },
   ];
 
+  const [showImageCropper, setShowImageCropper] = useState(false);
+  const [imageSrc, setImageSrc] = useState("");
+  const [fileName, setFileName] = useState("");
+
+  const convertBase64 = (file) => {
+    return new Promise((resolve, reject) => {
+      const fileReader = new FileReader();
+      fileReader.readAsDataURL(file);
+
+      fileReader.onload = () => {
+        resolve(fileReader.result);
+      };
+
+      fileReader.onerror = (error) => {
+        reject(error);
+      };
+    });
+  };
```

Comment:
sorry we are not using base64 anymore...

Reply:
This is for the image cropper component. Mentor image will be storing the cloudinary url.

Reply:
oh okayyy!!